### PR TITLE
autoformat victor.py

### PR DESCRIPTION
**Pull request by PeterMitrano**
also tweak gripper api for victor because I was too lazy to separate it. mea culpa.

### CODE REVIEW

File: arm_robots/src/arm_robots/victor.py [hunk 1]
```diff
@@ -19,16 +27,8 @@
     KUKA_FULL_SPEED_PATH_JOINT_IMPEDANCE_TOLERANCE, KUKA_GOAL_JOINT_IMPEDANCE_TOLERANCE
 from arm_robots.robot import MoveitEnabledRobot
 from arm_robots.robot_utils import make_joint_tolerance
-from control_msgs.msg import FollowJointTrajectoryFeedback, FollowJointTrajectoryGoal, FollowJointTrajectoryResult
-from moveit_msgs.msg import DisplayRobotState
-from std_msgs.msg import String, Float32
-from trajectory_msgs.msg import JointTrajectoryPoint
 from victor_hardware_interface.victor_utils import get_control_mode_params, list_to_jvq, jvq_to_list, \
```
Comment (bsaund) [inline arm_robots/src/arm_robots/victor.py:30]: I'm assuming the PR for `is_gripper_closed` is coming?

Comment (PeterMitrano) [reply arm_robots/src/arm_robots/victor.py:30]: oh somehow that file did not get included... one sec

Comment (PeterMitrano) [reply arm_robots/src/arm_robots/victor.py:30]: oh duh it's in the other repo. yes that PR is coming

Comment (PeterMitrano) [reply arm_robots/src/arm_robots/victor.py:30]: https://github.com/UM-ARM-Lab/kuka_iiwa_interface/pull/138

**Review by bsaund — APPROVED** [review]

**Pull request merged**